docs/source/integrations/triton.rst

```diff
@@ -1,3 +1,599 @@
-======
-Triton
-======
\ No newline at end of file
+=======================
+Triton Inference Server
+=======================
+
+*time expected: 12 minutes*
+
+NVIDIA Triton Inference Server [#triton]_ is a high performance, open-source inference server for serving deep learning models.
+It is optimized to deploy models from multiple deep learning frameworks, including TensorRT,
+TensorFlow, ONNX, to various deployments target and cloud providers. Triton is also designed with 
+optimizations to maximize hardware utilization through concurrent model execution and efficient batching strategies.
+
+BentoML now supports running Triton Inference Server through the :ref:`Runner <concepts/runner:Using Runners>`
+architecture. The following integration guide makes the assumption that readers are familiar with BentoML infrastructure.
+Make sure to check out our :ref:`tutorial <tutorial:Creating a Service>` should you wish to learn more about BentoML.
+
+The guide will try to be as comprehensive and detailed as possible, yet we will not be able to cover all the
+features from Triton Inference Server. For more information, please refer to their documentation [#triton_docs]_.
+
+The code examples in this guide can also be found in our examples folder [#triton_runner]_.
+
+
+Prerequisites
+~~~~~~~~~~~~~
+
+Make sure to have at least BentoML 1.0.14 and ``tritonclient`` at least version 2.29.0 available in your Python environment:
```

Comment:
Should "tritonclient" be installed as a plugin with "bentoml[triton]"?

Reply:
sure, but I think it will be come bloated for all of the features we want to add for future integration. It is ok to ask them to install addition deps, similar to spark.

Reply:
I have this discussion with Sauyon earlier today.

For integration such as Spark, users are required to have pyspark and spark available in order to use the features (or the user code is required to have pyspark in order to work).

For Triton, it is not required for users to have triton for their code to run. This means adding `bentoml[triton]` is ok in this case.

I will add a `bentoml[triton]` with this PR.

Reply:
Also add to bentofile.yaml.